node_helper.js

```diff
@@ -26,9 +28,13 @@ module.exports = NodeHelper.create({
 
 				this.sendSocketNotification('LIST_DATA', this.anylist.getListByName(list));
 			}
-
-			callback();
```

Comment:
It turns out that invoking this callback function always produces an error because we're calling it from inside an `async` function.  Adding the error logging exposed this.

According to the docs for the `async` node library, async queue workers don't use a callback function:
https://github.com/caolan/async/issues/1901